<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="dirLine">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>darknet Klasör Yolu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirBtn">
       <property name="text">
        <string>Klasör seç..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="configLine">
       <property name="placeholderText">
        <string>darknet Config Dosyası</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configBtn">
       <property name="text">
        <string>Dosya seç..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="weightsLine">
       <property name="placeholderText">
        <string>darknet Weights Dosyası</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="weightsBtn">
       <property name="text">
        <string>Dosya seç..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="dataLine">
       <property name="placeholderText">
        <string>darknet Data dosyası</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataBtn">
       <property name="text">
        <string>Dosya seç..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="kaydetBtn">
     <property name="text">
      <string>Kaydet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
